__tests__/commands/addTag.ts

```diff
@@ -0,0 +1,21 @@
+import { ApplicationCommandOptionTypes } from 'discord.js/typings/enums';
+import { execute } from '../../commands/addTag';
+import { setup } from '../../test-utils/setup';
+import { server } from '../../test-utils/test-server';
+
+describe('Command add-tag', () => {
+  it('should not run if tag is null', async () => {
+    const { createCommandInteraction } = await setup();
+    const interaction = createCommandInteraction(
+      'add-tag',
+      [{ name: 'tag', type: ApplicationCommandOptionTypes.STRING, value: null }],
+    );
+
+    // Instead of mocking this function, we could (should?)
+    // add a route to the discord handlers.
```

Comment:
Since I don't have a ton of testing experience I don't know how big of a lift this is.

If it's out of scope for this PR, could we create an issue on it so someone else might be able to pick it up?

Reply:
To solve just that one line should not be that big in scope, I didn't do it for 2 reasons, (1) I had been battling with Discord TypeErrors and I wanted to push the work forward, and (2) I'd say the work should be to add other test scenarios as well, not just that one. 

But I don't mind solving that one to keep the PR cleaner.

Reply:
Done!